zds/member/models.py

```diff
@@ -360,6 +362,12 @@ def get_followed_topics(self):
         return Topic.objects.filter(topicfollowed__user=self.user)\
             .order_by('-last_message__pubdate')
 
+    def is_dev(self):
+        """
+        Check if the user is in a group contained in settings.ZDS_APP['member']['dev_group'].
```

Comment:
Environ. Disons que ça passerait pour n'importe quel 'groupe' de choses, genre une liste, un set, voire un dict ou autres. Par contre ici on a clairement affaire à une string. `settings.ZDS_APP['member']['dev_group']` est une string. Et on va chercher un exact match, on va pas regarder si le groupe de l'utilisateur est `in` cette string.